dashboard/app/models/email_preference.rb

```diff
@@ -27,7 +27,7 @@ class EmailPreference < ApplicationRecord
 
   validates_presence_of :email, :ip_address, :source
   validates_inclusion_of :opt_in, {in: [true, false]}
-  validates_uniqueness_of :email
+  validates_uniqueness_of :email, case_sensitive: true
```

Comment:
This is another place where we are enforcing case-sensitive email uniqueness when we may not want to